myosuite/envs/myo/myobase/walk_v0.py

```diff
@@ -208,6 +212,10 @@ def _setup(self,
         self.init_qpos[:] = self.sim.model.key_qpos[0]
         self.init_qvel[:] = 0.0
 
+        # move heightfield down if not used
+        self.sim.model.geom_rgba[self.sim.model.geom_name2id('terrain')][-1] = 0.0
```

Comment:
I vaguely remember [-1] indexing caused some issue at some point. Not sure what the details were. 
@cameronberg @camberg23 -- do you remember?

Reply:
I don't recall if there was an issue here—what would the alternative be?

Reply:
Probably to assign sth like
`array[:]= [0,0,0]`

Reply:
Can this be resolved? @vikashplus